plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -431,3 +431,29 @@ def test__project_dir_does_not_exist_error(dbt_templater, caplog):  # noqa: F811
         ) in caplog.text
     finally:
         logger.propagate = original_propagate_value
+
+
+@pytest.mark.parametrize(
+    ("model_path", "var_value"),
+    [
+        ("models/vars_from_cli.sql", "expected_value"),
+        ("models/vars_from_cli.sql", [1]),
+        ("models/vars_from_cli.sql", {"nested": 1}),
+    ],
+)
+def test__context_in_config_is_loaded(
+    project_dir, dbt_templater, model_path, var_value  # noqa: F811
+):
+    """Test that variables inside .sqlfluff are passed to dbt."""
+    context = {"passed_through_cli": var_value} if var_value else {}
+
+    config_dict = DBT_FLUFF_CONFIG
+    config_dict["templater"]["dbt"]["context"] = context
```

Comment:
This looks like a possible bug in the test -- I think this code is actually making changes to `DBT_FLUFF_CONFIG`, which could break other tests. Please use `copy.deepcopy()` to copy the dictionary before making any changes to it.